src/umbraco.cms/businesslogic/web/DocumentType.cs

```diff
@@ -538,7 +543,10 @@ protected void PopulateDocumentTypeNodeFromReader(IRecordsReader dr)
         protected override void setupNode()
         {
             var contentType = ApplicationContext.Current.Services.ContentTypeService.GetContentType(Id);
-            SetupNode(contentType);
+            
+            // If it's null, it's probably a folder
+            if (contentType != null)

```

Comment:
Needed so that when this code is ran on a folder it doesn't crash. When it's not an actual document type `SetupNode` should be skipped.